### DISCUSSION TITLE
No longer working?

### DISCUSSION THREAD

**Discussion by EurikaDude** [zerodytrash/Simple-YouTube-Age-Restriction-Bypass#248] [Q&A] [answered]
Is this just something on my end or is the script in need of an update?

**Comment by drunkwinter** [comment, marked as answer]
Hi, please try the test builds posted in issue #247 and let me know if one of those fixes it.

I need some more feedback before releasing.

**Reply by EurikaDude** [reply]
Ah yes, thank you! Build 2 in that thread worked for me

**Reply by drunkwinter** [reply]
Thank you for the feedback
